taccsite_cms/settings.py

```diff
@@ -375,52 +357,6 @@ def get_subdirs_as_module_names(path):
     'easy_thumbnails.processors.filters'
 )
 
-
-# FEATURES.
-if CONSOLE_LOG_ENABLED:
-    print("--> Variable FEATURES: ")
-    for feature in FEATURES:
-        print(feature + ": ", FEATURES[feature])
-
-if current_secrets._FEATURES['blog']:
-    # Install required apps
-    INSTALLED_APPS += [
-        # Blog/News
-        # 'filer',              # Already added
-        # 'easy_thumbnails',    # Already added
-        'parler',
-        'taggit',
-        'taggit_autosuggest',
-        'meta',                 # also supports `djangocms_page_meta`
-        'sortedm2m',
-        'djangocms_blog',
-
-        # Metadata
-        'djangocms_page_meta',
-    ]
-
-    # Metadata: Configure
-    META_SITE_PROTOCOL = 'http'
-    META_USE_SITES = True
-    META_USE_OG_PROPERTIES = True
-    META_USE_TWITTER_PROPERTIES = True
-    META_USE_GOOGLEPLUS_PROPERTIES = True # django-meta 1.x+
-    # META_USE_SCHEMAORG_PROPERTIES=True  # django-meta 2.x+
-
-    # Blog/News: Set custom paths for templates
-    BLOG_PLUGIN_TEMPLATE_FOLDERS = (
-        ('plugins/default', 'Default template'),    # i.e. `templates/djangocms_blog/plugins/default/`
-        ('plugins/default-clone', 'Clone of default template'),  # i.e. `templates/djangocms_blog/plugins/default-clone/`
-    )
-
-    # Blog/News: Change default values for the auto-setup of one `BlogConfig`
-    # SEE: https://github.com/nephila/djangocms-blog/issues/629
-    BLOG_AUTO_SETUP = True
-    BLOG_AUTO_HOME_TITLE ='Home'
-    BLOG_AUTO_BLOG_TITLE = 'News'
-    BLOG_AUTO_APP_TITLE = 'News'
-
-
```

Comment:
⚠️ This change removes a large feature that is already installed on Texascale.

1. What is the plan for supporting this feature per site without duplicate setup?
2. What is the plan for preventing failure on Texascale when it loads this change?

<details><summary>Notes</summary>

- This feature is planned to be used for TACC, also.
- This is related to removal of a sub-feature in [`urls.py`](https://github.com/TACC/Core-CMS/pull/341/files#diff-ed9c8319f3a9f96a3b5db913871c815cf80ff229a6134b3655396a1e0a7e2701)

</details>